httpcore5/src/main/java/org/apache/hc/core5/http/io/entity/EntityUtils.java

```diff
@@ -55,7 +55,12 @@
  */
 public final class EntityUtils {
 
+    private static final Charset DEFAULT_CHARSET = StandardCharsets.ISO_8859_1;
```

Comment:
I wonder why are still have this. RFC 7230 and friends to not define any default encoding any more.